.circleci/config.yml

```diff
@@ -148,6 +168,19 @@ jobs:
       - run: *step-test-func
       - run: *step-verify-git-clean
 
+  verify-windows:
+    executor:
+      name: win/default
+    steps:
+      - run: *step-configure-git-on-windows
+      - checkout
+      - run: *step-install-yarn-on-windows
+      - run:
+          name: Install Node modules
+          command: yarn
+      - run: *step-build
+      - run: *step-verify-git-clean
```

Comment:
Executor image is not pinned and can be updated by third party. It's ok for windows image, but I propose to add NVM step to pin Node version. (NVM is already installed).